src/Calculator/Views/GraphingCalculator/EquationInputArea.xaml.cpp

```diff
@@ -108,6 +162,10 @@ void EquationInputArea::EquationTextBox_RemoveButtonClicked(Object ^ sender, Rou
             m_lastFunctionLabelIndex--;
         }
 
+        if (index == Equations->Size - 1 && Equations->Size > 1)
+        {
+            Equations->GetAt(Equations->Size - 2)->IsLastItem = true;
```

Comment:
Is IsLastItem the last item in the list of equations or the last item to have focus?

Reply:
Maybe adjust the name so that it is more clear.

Reply:
changed! `IsLastItemInList`